index.bs

```diff
@@ -115,6 +118,9 @@ The <dfn for="a" element-attr>attributionreportto</dfn> attribute optionally dec
 The <dfn for="a" element-attr>attributionexpiry</dfn> attribute optionally defines the amount
 of time in milliseconds the attribution source should be considered for reporting.
 
+The <dfn for="a" element-attr>attributionsourcepriority</dfn> attribute optionally defines the
+priority of a source relative to other sources when triggering attribution.
```

Comment:
We should say something here about the meanings of the possible values. Will an attribution with a higher priority value be picked instead of one with a lower value? What's the default? Do different mutually-trusting parties need to agree at all about their prioritization scheme (including different libraries used by the same company), and if so, what kind of agreement do we suggest?

Reply:
Added a bit more color. The actual behavior is defined by the attribution triggering algorithm which is why I left out more detail here, but having more detail next to the definition seems valuable.

Priorities are interpreted within <reporting endpoint, attribution destination> scopes, so generally third parties will not need to agree on this. If there are libraries using the same reporting endpoint, they will need to agree.